test/specs/mainnet/01_TransferringFunds/Optimism.spec.js

```diff
@@ -71,7 +72,13 @@ describe('The PrimeSDK, when transfer a token with optimism network on the MainN
       );
     }
 
-    let output = await optimismMainNetSdk.getAccountBalances({
+    // initializating Data service...
+    optimismDataService = new DataUtils(
```

Comment:
Can we initialise this on the BeforeAll block so that you don't have to initialise every time on the test cases?

Reply:
It's already added in the BeforeEach block.